bitrise.yml

```diff
@@ -176,6 +176,23 @@ workflows:
         - BITRISE_TEST_SHARDS_PATH: BITRISE_TEST_SHARDS_PATH_FIREFOX 
     - save-spm-cache@1.3.1:
         is_always_run: true
+    - script@1.1.1:
+        title: Clean before deploying
+        timeout: 360
+        inputs:
+        - content: |
+            #!/usr/bin/env bash
+            set -euo pipefail
+
+            echo "Removing raw-xcodebuild-output.log to reduce artifact size"
+            rm -f "$BITRISE_DEPLOY_DIR/raw-xcodebuild-output.log"
+
+            echo "Keeping only one swiftlint report file"
+            # Remove one format or duplicate copies
+            rm -f "$BITRISE_DEPLOY_DIR/swiftlint_report.txt" || true
+
+            echo "Removing unit test xcresult"
+            rm -f "$BITRISE_DEPLOY_DIR/Test-Fennec_UnitTest_iphonesimulator18.2-arm64.xcresult.zip"
```

Comment:
Yes, it's a good idea to upload only the files we need. How much time/space do we save by removing those files? I choose making `bitrise.yml` simple to read rather than having all the small optimizations.

Reply:
Not sure yet how much we save, will need a few runs to compare or lands and see the improvements if any. The deploy step takes ~1.5mins, I would say all the optimizations are good even though they are small